tlstunnel.install

```diff
@@ -0,0 +1,6 @@
+bin: [
+  "_build/src/tlstunnel.native" {"tlstunnel"}
```

Comment:
fixed (although bytecode only still wont work due to man page generation)

> On 16 Apr 2015, at 09:27, Thomas Gazagnaire <notifications@github.com> wrote:
> 
> In tlstunnel.install <https://github.com/hannesm/tlstunnel/pull/12#discussion_r28491808>:
> 
> > @@ -0,0 +1,6 @@
> > +bin: [
> > +  "_build/src/tlstunnel.native" {"tlstunnel"}
> In case the native compiler is not available on the platform it's better to have:
> 
> "?_build/src/tlstunnel.byte" {"tlstunnel"}
> "?_build/src/tlstunnel.native" {"tlstunnel"}
> —
> Reply to this email directly or view it on GitHub <https://github.com/hannesm/tlstunnel/pull/12/files#r28491808>.
>